### ISSUE TITLE
buid failure (with ./configure --with-libgcrypt && make)

### ISSUE THREAD

**Issue by citrin** [libssh2/libssh2#150]
I'm trying to build libssh 1.8.0 and
```
./configure --with-libgcrypt
make
```
fails with an error
```
  CCLD     libssh2.la
../libtool: line 7474: cd: yes/lib: No such file or directory
libtool:   error: cannot determine absolute directory name of 'yes/lib'
Makefile:531: recipe for target 'libssh2.la' failed
make[2]: *** [libssh2.la] Error 1
```
This problem seems to be appeared after commit 42941b44f8c9ce79588ea22e4b9bf221c7019285 in which "yes" is added to CFLAGS and LDFLAGS (probably value from --with-libgcrypt-prefix should be used instead --with-libgcryt).

**Comment by alamaison** [comment]
Some changed have just been merged into master that fix some issues with configure.  Can you try the latest master?

**Comment by citrin** [comment]
On fresh master it works. As I can see it was fixed in ced924b78a40126606797ef57a74066eb3b4b83f.
